packages/docs/src/routes/tutorial/projection/slots/index.mdx

```diff
@@ -6,7 +6,7 @@ In simple cases, projection allows content from the parent component to be proje
 
 ### Example
 
-In this example, we have created `<Collapsable>` component that toggles between open and closed states. Currently, when the `<Collapsable>` is closed it does not show any content. Add additional `<Slot>` to project the `q:slot="closed"` content.
+In this example, we have created `<Collapsable>` component that toggles between open and closed states. Currently, when the `<Collapsable>` is closed it shows its own content. Change that to an additional `<Slot>` to project the `q:slot="closed"` content from the parent instead.
```

Comment:
Thanks!
I'm not sure I get "it shows its own content"

Can you clarify please?